src/index/generateTrees/toFractalTree.ts

```diff
@@ -116,22 +127,22 @@ export function toFractalTree(graph: Graph, entryPoints: string[]) {
         continue;
       }
 
-      const testFilePath = res[firstRelativeImport];
-      if (testFilePath) {
-        let location = path.join(
-          path.dirname(testFilePath),
-          path.basename(testFile)
-        );
+      const testFilePath = tree[firstRelativeImport];
 
-        location = checkDuplicates(
-          location,
-          path.dirname(testFilePath),
-          testFile
-        );
-        res[testFile] = location;
+      if (!testFilePath) {
```

Comment:
I'd prefer us to be explicit in cases like this. It can also be converted into a one-liner as it escapes.

Reply:
Sorry, I have trouble understanding this comment. Could you give me an example?

Reply:
https://github.com/benawad/destiny/pull/89#discussion_r386655012